.github/workflows/release-prod.yml

```diff
@@ -14,7 +14,9 @@ jobs:
   release:
```

Comment:
This was renamed to `release.yml` on `main`. Left it as-is to make it easier to see the changes. Will create a follow-up PR, to rename it.